pkgs/top-level/all-packages.nix

```diff
@@ -12974,6 +12974,7 @@ with pkgs;
     callPackage = newScope self;
 
     inherit kernel;
+    stdenv = overrideCC pkgs.stdenv kernel.buildCc;
```

Comment:
@aszlig: I switched to the form you suggested. I also squashed commit 1+2 together as commit 2 became trivial.